client/verta/verta/_deployment/endpoint.py

```diff
@@ -29,10 +31,15 @@ def _create(cls, conn, conf, workspace, path, description=None):
 
     @classmethod
     def _create_json(cls, conn, workspace, path, description=None):
+        json = {}
+        if description:
+            json["description"] = description
         if not path.startswith('/'):
             path = '/' + path
-
-        raise NotImplementedError
+        json["path"] = path
+        response = _utils.make_request("POST", "{}://{}{}".format(conn.scheme, conn.socket, "/api/v1/deployment/workspace/{}/endpoints".format(workspace)), conn, json=json)
```

Comment:
Moving the URL to a separate line will make it easier to read
```suggestion
        url = "{}://{}/api/v1/deployment/workspace/{}/endpoints".format(conn.scheme, conn.socket, workspace)
        response = _utils.make_request("POST", url, conn, json=json)
```